unified-codes/apps/web/src/actions/explorer/searchBar.ts

```diff
@@ -0,0 +1,33 @@
+import { Action } from 'redux';
+
+export const SEARCH_BAR_ACTIONS = {
+  UPDATE_INPUT: 'explorer/searchBar/updateInput',
+  UPDATE_LABEL: 'explorer/searchBar/updateLabel',
+};
```

Comment:
Note that the redux shape mirrors the component hierarchy. Makes for a tidy single source of truth and should make it a lot more straightforward to hook up components with state/logic!